cql3/values.cc

```diff
@@ -58,4 +58,42 @@ raw_value_view::raw_value_view(managed_bytes&& tmp) {
     _data = managed_bytes_view(*_temporary_storage);
 }
 
+raw_value make_collection_raw_value(size_t size_to_write, const std::vector<raw_value>& elements_to_write, cql_serialization_format sf) {
```

Comment:
An even better option is to move this function to the test source file. It doesn't need to be a general utility function, and putting it in values.cc won't mean it is magically maintained by anyone.

Personally, I wouldn't have done this in C++ at all, and do it in cql-pytest (but I didn't check - it is possible that the Python driver "protects" you from sending garbage, so it's not possible).

Reply:
I originally put in the test source, but it's very generic so I thought that maybe it's better to put it somewhere global so that it can be used in other places in the future.
I don't have any strong opinion either way, I will move it to the test file.

Reply:
> Personally, I wouldn't have done this in C++ at all, and do it in cql-pytest (but I didn't check - it is possible that the Python driver "protects" you from sending garbage, so it's not possible).

It prevents me from sending invalid values
Trying to send `[1, None, 2]` results in a list with empty value instead of null.
Trying to send `[1, UNSET_VALUE, 2]` Fails before query even leaves the driver.

This test looks good when written using the rust driver. It allows to create collections easily just like python, but doesn't have any protections and obediently serializes whatever it's given. Alas we can't write tests using rust yet.